src/main/java/core/basesyntax/FigureInformationPrinting.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface FigureInformationPrinting extends AreaCalculator {

```

Comment:
```suggestion
public interface drawable {
```
we can obtain the area of any figure and are able to draw it.  - We should be able to draw **figure** - not area.

Reply:
not fixed